README.md

```diff
@@ -191,7 +191,7 @@ If you would you like to start your own language implementation of IPFS, check o
 | C++ | https://github.com/vasild/cpp-ipfs-api | |
 | Julia | contact: [@rened](https://github.com/rened) | 0% |
 | Lua | contact: [@seclorum](https://github.com/seclorum) | 0% |
-| Erlang | ! | 0% |
+| Erlang | https://github.com/hendry19901990/erlang-ipfs-api |  |
```

Comment:
That's awesome @hendry19901990 ! Thank you for making one.

What's the state of it? Does it implement all the calls that https://github.com/ipfs/js-ipfs-api does? Is it compatible with all go-ipfs versions?

Also, I've noticed there are no docs, it will be hard for people to use it without them, do you have a plan of adding?